src/yea_wandb/backend.py

```diff
@@ -131,17 +139,18 @@ def reset_ctx():
                 else:
                     raise ValueError("Server failed to start.")
         if started:
-            print(
-                "INFO: Mock server listing on {} see {}".format(server._port, logfname)
-            )
+            print(f"INFO: Mock server listing on {server._port}, see {logfname}")
         else:
             server.terminate()
-            print("ERROR: Server failed to launch, see {}".format(logfname))
+            print(f"ERROR: Server failed to launch, see {logfname}")
             raise Exception("problem")
 
         os.environ["WANDB_BASE_URL"] = f"http://{mockserver_host}:{port}"
         if not mockserver_relay:
             os.environ["WANDB_API_KEY"] = DUMMY_API_KEY
+        else:
+            # os.environ["WANDB_BASE_URL"] = mockserver_relay_remote_base_url
```

Comment:
@raubitsj a question regarding this: this whole thing works fine, but the URLs that get printed in the terminal look like this:
<img width="612" alt="image" src="https://user-images.githubusercontent.com/7557205/167732301-d8f84daf-6b0b-41fc-b98e-82dc3d6b7903.png">
If you replace the base url with the cloud base url -- those are actually the correct links, but if I change `WANDB_BASE_URL` here, the correct url get printed, but the subsequent context enrichment fails (nothing gets propagated). What am I missing here?